test/spec/protractor_sync_sherman_test.ts

```diff
@@ -0,0 +1,49 @@
+//TODO:  This is a temporary test file to be removed before merged.
+//TODO: The contents of this file should move ino README.md when ready
+import * as ab from 'asyncblock';
+
+import * as protractorSync from '../../app/index';
+import { browserSync, elementSync, polledExpect} from '../../app/index';
```

Comment:
minor: are the imports here written kinda funny? You imported everything as protractorSync at line 5, but then you also imported specific methods at line 6. Is there a reason for that, or could we simplify by just importing all required methods by name in line 6?

Reply:
good catch! updated and verified.  
Removes the ```import *``` and adds ```configure``` to the mix